cardstack/templates/page-welcome.php

```diff
@@ -0,0 +1,163 @@
+                    "_" . $_SERVER['REMOTE_ADDR'];
+                $cardstack_am_stream_token = CardStackAm::obfuscateString($cardstack_am_pure_stream_str);
+                if ($_GET["altsub"] === "yes") {
+                    $cardstack_am_episode = $cardstack_am_episode . 'a';
+                }
+                $cardstack_am_video = CardStackAmConstants::getVidUrl() .
+                    "stream/film_stream.php/Chuuni" . $cardstack_am_episode . ".webm?token=" .
+                    $cardstack_am_stream_token;
+                $cardstack_am_poster = "https://static.animagia.pl/film_poster.jpg";
+
+                if (IP_Geo_Block::get_geolocation()['code'] !== 'PL') {
+                    $cardstack_am_video = "";
+                }
+
+                if (CardStackAm::userCanStreamProduct(318)) :
+
+                    if ($cardstack_am_episode == "00") {
+                        echo '<p>Jeśli wolisz napisy bez japońskich tytułów grzecznościowych, przejdź <a href="'
+                            . get_home_url() . '/?altsub=yes">tutaj</a>.</p>';
+                    } else if ($cardstack_am_episode == "00a") {
+                        echo '<p>Napisy bez japońskich tytułów grzecznościowych, z zachodnią kolejnością imion i nazwisk.</p>';
+                    }
+                    ?>
+
+                    <!--data-setup='{"playbackRates": [1, 1.1, 1.2, 2] }'-->
+                    <video id='amagi' class="video-js vjs-16-9 vjs-big-play-centered" style="width: 100%;"
+                           controls="true" oncontextmenu="return false;"
+                           poster="<?php echo $cardstack_am_poster ?>" preload="metadata"
+                           data-setup='{}'>
+                        <source src="<?php echo $cardstack_am_video ?>" type="video/webm" />
+                    </video>
+
+                    <script src="https://static.animagia.pl/video.js"></script>
+
+                    <script>
+                        var vid1 = videojs('amagi');
+                        vid1.on('dblclick', function () {
+                            vid1.requestFullscreen();
+                        });
+                        vid1.on('dblclick', function () {
+                            vid1.exitFullscreen();
+                        });
+                    </script>
+
+                <?php
+                else :
+
+                    CsAmVideo::printFreeFilmPlayer(); ?>
+
+
+                <?php endif; ?>
+
+                <?php comments_template(); ?>
+
+            <?php endwhile; ?>
+        <?php endif; ?>
+
+    </article>
+
+    <article class="page<?php
+    if (get_theme_mod('page_breadcrumbs')) {
+        print(' has-breadcrumbs');
+    }
+    ?>">
+        <div class="container">
+            <?php
+            print("<img src=".get_template_directory_uri()."/templates/Elementario.png alt=\"Amagi\"/>")
+            ?>
+            <div class="image-text">
+                <p>
```

Comment:
Layout seems to break on smaller screens, when the text doesn't fit in the image. Text should be clipped in such a case.